.github/workflows/ci-e2e.yml

```diff
@@ -188,7 +188,7 @@ jobs:
                   DISABLE_SECURE_SSL_REDIRECT=1
                   SECURE_COOKIES=0
                   OPT_OUT_CAPTURE=0
-                  SELF_CAPTURE=0
+                  SELF_CAPTURE=1
```

Comment:
Actually, now I wonder if this is going to cause any issues.... I know we currently capture cypress flows in a project in posthog... Maybe verify in #ask-posthog-anything?